src/main/java/dev/galacticraft/api/universe/celestialbody/landable/Landable.java

```diff
@@ -33,7 +32,7 @@
  *
  * @param <C> the type of configuration
  */
-public interface Landable<C extends CelestialBodyConfig> extends SurfaceEnvironment<C> {
+public interface Landable<C extends CelestialBodyConfig> {
```

Comment:
Why did you remove the `SurfaceEnvironment` extension? All subtypes of `Landable` have this interface directly applied now (which kind of defeats the purpose)

Reply:
Because things that are not `Landable` have the `SurfaceEnviornment` extension as well. (Decorative planets and Decorative stars)